ci/codec_cpu/audit.py

```
@@ -0,0 +1,1253 @@
+    return report
+
+
+def parse_callgrind(path: Path) -> CallgrindData:
+    events: list[str] = []
+    summary: dict[str, int] = {}
+    for line in path.read_text(encoding="utf-8", errors="replace").splitlines():
+        if line.startswith("events:"):
+            events = line.split()[1:]
+        elif line.startswith("summary:") and events:
+            values = [int(value) for value in line.split()[1:]]
+            summary = dict(zip(events, values))
```

Ruff (0.16.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)